src/theory/strings/theory_strings_utils.h

```diff
@@ -153,6 +153,14 @@ bool isStringKind(Kind k);
  */
 TypeNode getOwnerStringType(Node n);
 
+/* Get the number of repetitions for a regexp repeat node */
+unsigned getRepeatAmount(TNode node);
+
+/* Get the upper index of given regexp loop node. */
+unsigned getLoopHigh(TNode node);
+/* Get the lower index of given regexp loop node. */
+unsigned getLoopLow(TNode node);
```

Comment:
```suggestion
/* Get the number of repetitions for a regexp repeat node */
uint32_t getRepeatAmount(TNode node);

/* Get the upper index of given regexp loop node. */
uint32_t getLoopHigh(TNode node);
/* Get the lower index of given regexp loop node. */
uint32_t getLoopLow(TNode node);
```

Reply:
I am not quite sure about the naming of `getLoopHigh`/`getLoopLow`. What do you think about `getLoopMaxOccurrences`/`getLoopMinOccurrences` or `getLoopMaxAmount`/`getLoopMinAmount` (and renaming the corresponding members of the operator accordingly)?

Reply:
Sure.